src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFilterTypes.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using System.Text;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement
+{
+    /// <summary>
+    /// Types of built-in feature filters.
+    /// </summary>
+    internal class FeatureFilterTypes
```

Comment:
Based off the usage, this looks like `FeatureFilterConfiguration` or `FeatureFilterSetup`

Reply:
Or even FeatureFilterTelemetry